src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -234,6 +243,13 @@ public with sharing class ADDR_Addresses_TDTM extends TDTM_Runnable {
 	                }      
 	            }
             }
+            
+            // finally, if no default found, create an empty default
```

Comment:
Wouldn't it be better to clear them another way, so that we don't have empty address objects hanging around eating up data storage?